src/status_im/contexts/shell/jump_to/components/switcher_cards/style.cljs

```diff
@@ -1,6 +1,10 @@
 (ns status-im.contexts.shell.jump-to.components.switcher-cards.style
   (:require
-    [quo.foundations.colors :as colors]))
+    [quo.foundations.colors :as colors]
+    [react-native.core :as rn]))
+
+(def screen-width (:width (rn/get-window)))
+(def card-size (- (/ screen-width 2) 28))
```

Comment:
hi @mohsen-ghafouri, Thank you very much for fixing this issue. 

- Please can you elaborate this calculations?
- Also can we pass, screen-width or card-size in function itself. (I have seen some bugs where calling rn function for getting info like dimensions at start of app returned nil or wrong value)

Reply:
Hi @Parveshdhull, I calculate the width by dividing the total width by 2 and subtracting the space from it, as the list has 2 columns.

At first, I used `width (rf/sub [:dimensions/window-width])` in the parent function, but I thought it an unnecessary dependency for the child component. If (rn/get-window) isn't consistently reliable, I'll opt to use `:dimensions/window-width` in the main function instead.

Reply:
but if the issue is only happens at start of app, here in shell should be safe, no?

Reply:
> subtracting the space from it

Please can you share figma screenshot which space. Sorry, I was having hard time finding 28 gap.

> 'll opt to use :dimensions/window-width in the main function instead.

you can use still use `(rn/get-window)` in the main function.

> but if the issue is only happens at start of app, here in shell should be safe

We are defining these using def, they will be called at the start of app.

Reply:
@Parveshdhull could you please check again? i also made calculation more clear